templates/rancher-backup/0/README.md

```diff
@@ -0,0 +1,52 @@
+# Rancher Backup
+
+## Info:
+
+This template deploy one container that manage backup aspect in your Rancher environment.
+For the moment, this template permit to perform the following actions:
+- Dump automatically all databases supported before start the remote backup:
+  - PostgreSQL
+  - MySQL
+  - MariaDB
+  - MongoDB
+  - Elasticsearch
+ - Dump automatically all rancher stacks settings (docker-compose.yml and rancher-compose.yml) before start the remote backup
+ - Perform the remote backup of all `/backup` with duplicity (you can mount your shares storages on `/backup` to backup them)
+
+ To manage your backup policy, you can use the following environment variables:
+ - `CRON_SCHEDULE`: when you should start backup (incremental if full is not needed). For example, to start backup each day set `0 0 0 * * *`
+ - `BACKEND`: this is the target URL to externalize the backup. For example, to use FTP as external backup set `ftp://login@my-ftp.com` and add environment variable `FTP_PASSWORD`. For Amazon S3, set `s3://host[:port]/bucket_name[/prefix]`. Read the ducplicity man for [all supported backend](http://duplicity.nongnu.org/duplicity.1.html#sect7). There are no default value.
```

Comment:
I have added 2 new entries to set the AWS key and AWS secret. Then you must use the backend url `s3://host[:port]/bucket_name[/prefix] `

(I have tested with minio, all works fine)